src/models/AppState.js

```diff
@@ -586,6 +586,15 @@ function (Okta, Q, Factor, BrowserFeatures, Errors) {
         fn: function (policy) {
           return policy && policy.rememberDeviceByDefault;
         }
+      },
+      'factorsPolicyInfo' : {
+        deps: ['policy'],
+        fn: function (policy) {
+          if(policy && policy.factorsPolicyInfo) {
```

Comment:
Nit: Can just be something like: return ```(policy && policy.factorsPolicyInfo) ? policy.factorsPolicyInfo: false;```

Reply:
Also, since `factorsPolicyInfo` is an object, maybe better to return a `null` rather than `false` here in case it does not exist?